mvc/views/tag-helpers/authoring/sample/TagHlp/src/AuthoringTagHelpers/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,15 @@
+﻿@using AuthoringTagHelpers.Models
+@{
+    ViewData["Title"] = "Home Page";
+}
+
+<div>
+    <h3>Information about our website (outdated):</h3>
+    <Website-InforMation info=@Model />
```

Comment:
You should also specify the `@model` at the top of the page instead of relying on dynamic.